app/components/Button/index.jsx

```diff
@@ -0,0 +1,38 @@
+import cx from 'classnames'
+import React from 'react'
```

Comment:
Changes in https://github.com/zanata/zanata-spa/pull/97 that I can merge here when this is finished reviewing.